.gitignore

```diff
@@ -21,9 +21,6 @@ local.properties
 .classpath
 .project
 
-# Windows thumbnail db
-.DS_Store
```

Comment:
Good catch! and I prefer It looks better to replace line with `Thumbs.db` instead of just deleting it. 
Could you please do it?

Reply:
Gotcha, I'm not good at Windows... so much helpful for me... 🙇

Reply:
Fixed in 30589d54fb12e38d85345cfe5bc398182da7c7d7